src/eval-frame/components/reps/tiny-reps.jsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
```

Comment:
this entire fire could use prop types on all of these functional components. In cases where you have a factory, the functional component the factory creates could easily get its proptypes also assigned

Reply:
I might also suggest calling this file `tiny-rep.jsx` since that's the thing we're technically exporting from it

Reply:
:+1: